### DISCUSSION TITLE
Testing Release 4.0.0 - https://eclipse.gwtproject.org.vertispan.com/2023-02-01

### DISCUSSION THREAD

**Discussion by protoism** [gwt-plugins/gwt-eclipse-plugin#442] [Releases]
I'm trying to find a reasonably friendly way to collect plugin testers feedback

### How to contribute

Reply to this thread with simple answers such as: 

O/S: ...
Eclipse version: ...
JDK version: ...
Status: {Working / Mostly working / Not working}
Notes: ....

### Plugin installation

The plugin must be installed via Help -> Installed Software.

First add an update site:

![image](https://user-images.githubusercontent.com/1098792/217497870-311aa936-5f90-465d-bf73-84ea05274edd.png)

Then install the plugin using the just added update site:

![image](https://user-images.githubusercontent.com/1098792/217498498-f4d60849-f5c4-4d49-8900-b5b0670ed412.png)

Note: the plugin is currently not signed, so Eclipse will ask for confirmation

### Verify that the plugin is correctly installed

After restart, the plugin should be visible in Help->Installation details

![image](https://user-images.githubusercontent.com/1098792/217499061-05cfd418-f924-41a5-b215-c337c767663c.png)

### Collect Eclipse / JDK versions

How to find Eclipse version
![image](https://user-images.githubusercontent.com/1098792/217494801-acea8c8a-e213-4666-9ab7-a55cbf0e5a88.png)

How to find JDK
![image](https://user-images.githubusercontent.com/1098792/217495615-bb1c210b-c81f-4dac-bdcd-56f2bccbbc2e.png)

**Comment by protoism** [comment]
O/S: Windows 11
Eclipse version: Eclipse 2022-12
JDK version: JDK 17
Status: Working

**Comment by axeluhl** [comment]
I installed this (GWT Eclipse Plugin	4.0.0.202302020242	com.gwtplugins.eclipse.suite.v3.feature.feature.group	GWT Eclipse Plugin) as an update from the update site over an installed 3.0.0:

Ubuntu 20.04.5 LTS
Eclipse version 2022-12 (Build id: 20221201-1913)
java.runtime.version=17.0.6+10 (Eclipse-provided plugins/org.eclipse.justj.openjdk.hotspot.jre.full.linux.x86_64_17.0.6.v20230125-1136/jre/bin/java)
Status: Working

**Comment by ishres19** [comment]
Eclipse: version: 2022-12 (4.26.0), Build id: 20221201-1913
MacOS version 12.5
OpenJDK 11.0.17
Status: Working!
GWT Plugin installed using: https://eclipse.gwtproject.org.vertispan.com/2023-02-01

**Comment by georgevaccaro** [comment]
OS: Ubuntu 22.04.1 LTS (GNU/Linux 5.15.0-58-generic x86_64) (xubuntu)
Eclipse Version: 2022-12 (4.26.0), Build id: 20221201-1913
eclipse.vm=/home/user/Development-local/eclipse//plugins/org.eclipse.justj.openjdk.hotspot.jre.full.linux.x86_64_17.0.5.v20221102-0933/jre/bin/java

I don't think it worked, though it didn't complain during installation.

![Screenshot_2023-02-09_09-15-33](https://user-images.githubusercontent.com/2207417/217839866-a34bb25b-4f43-405c-9ffe-f5439f271ff3.png)
![Screenshot_2023-02-09_09-15-55](https://user-images.githubusercontent.com/2207417/217839954-8ea1072d-d89e-45f5-a3e5-8296ef74df52.png)
![Screenshot_2023-02-09_09-22-58](https://user-images.githubusercontent.com/2207417/217840007-f34794df-09cb-4467-b9d6-7dce0c01d362.png)

Any ideas?

**Reply by georgevaccaro** [reply]
Oops - might be user error.  This signing window was somehow obscured during the installation process:

![Screenshot_2023-02-09_09-30-52](https://user-images.githubusercontent.com/2207417/217841800-6cd27bec-6b37-482b-96d1-88c16148f92d.png)

Now it's all showing up.

![Screenshot_2023-02-09_09-31-41](https://user-images.githubusercontent.com/2207417/217841821-7700b711-c1ed-457d-acf3-79418f8a9770.png)

![Screenshot_2023-02-09_09-33-10](https://user-images.githubusercontent.com/2207417/217842125-9421bc0c-6b8f-4022-85e5-74ce37ac9de2.png)

One note, if I hover over the GWT toolbar entry it says "GDT Pulldown":
![Screenshot_2023-02-09_09-33-27](https://user-images.githubusercontent.com/2207417/217842239-e07fb555-0839-4bb9-b7f5-5703f1bd6c5e.png)

**Reply by niloc132** [reply]
Thanks, bad tooltip filed as https://github.com/gwt-plugins/gwt-eclipse-plugin/issues/444

**Comment by niloc132** [comment]
Anyone else planning to test, or can we promote this to a release?
